lib/elements/text.js

```diff
@@ -155,20 +155,29 @@ class TextPrompt extends Prompt {
 
   render() {
     if (this.closed) return;
-    if (!this.firstRender) {
+    if (this.firstRender) {
+      // Make sure we're not on the very last line of the terminal.
+      // If a linefeed causes the terminal to scroll, the line count
+      // will not be accurate, which causes problems with counting lines
+      // backup up
+      this.out.write('\n\n' + cursor.up(2));
+      this.prevRowCount = 0;
```

Comment:
This attempts to account for the terminal being on the last row.  It's not going to work well if the prompt is multiple lines, if the hint/error is multiple lines, or if the input wraps, though.  For anything more complex... there needs to be a row check.

Reply:
Fixed